docs/docs/core/content-manager/review-workflows.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: Review Workflows
+slug: /content-manager/review-workflows
+description: Guide for review workflows in the content-manager.
+tags:
+  - content-manager
+  - review-workflows
+---
+
+## Summary
+
+Review workflows are disabled for all content-types by default and have to be enabled for each of them. More about how to [enable review-workflows for a content-type](/content-type-builder/review-workflows).
+
+The feature is visible in two locations of the content-manager:
+
+### List view
+
+If the feature is enabled for a content-type a new column will show up, displaying the current stage. If no stage was assigned to an entity,
+the column is displayed as empty.
+
+### Edit view
+
+If the feature is enabled for a content-type the currently selected stage will show up in the information sidebar next to the edit view. Users
+can select any other stage of the current workflow.
+
+Stage assignments are decoupled from entities, meaning that updating an entity won't set the selected stage. Instead the stage select
+component will trigger an atomic update in the admin API to assign/ update a stage to the current entity, when a new value is selected.
```

Comment:
```suggestion
component will trigger an automatic update using the admin API to assign/ update a stage to the current entity, when a new value is selected.
```

Reply:
or did you mean atomic? I guess it could be described as atomic...

Reply:
Yes I meant atomic :) I'll update in -> using